.github/workflows/deploy.yml

```
@@ -0,0 +1,47 @@
+      pages: write
+      id-token: write
+
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Deploy to GitHub Pages
+        id: deployment
+        uses: actions/deploy-pages@v4
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)